### ISSUE TITLE
Add alt text for og:image

### ISSUE THREAD

**Issue by bkmgit** [mmistakes/minimal-mistakes#3562]
### What happened?

- Minimal Mistakes version:
- Ruby gem or remote theme version:
- Jekyll version:
- Git repository URL:
- Hosted on GitHub Pages (if yes provide URL to site):
- Operating system:

### Expected behavior

An option to specify alt text for og:image would make these accessible to screen readers. See https://ogp.me/#structured

### Steps to reproduce the behavior

Information about creating a teaser image and logo is available at https://mmistakes.github.io/minimal-mistakes/docs/configuration/#site-default-teaser-image however https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/seo.html#L70 does not have alt text

### Other

This will complicate setting up multilingual sites, but may be done as a separate issue.
